### ISSUE TITLE
Discovery: Supplementary Reports/Corrective reports

### ISSUE THREAD

**Issue by suhafa** [bcgov/cas-reporting#176]
#### Goal of Ticket
Understand supplementary/corrective reports process

#### Describe the task

We need to know the process of supplementary reports, the difference between corrective and supplementary reports, how it was done in the past, how it affects all the moving pieces in the app, editing reports and how to maintain records of changes and so on.

Some open questions:
- For corrective reports, we know Adria would be involved. How do we want to handle this in the app?
- What have been some use cases for supplementary reports? What is the best that can happen? What is the worst? (legal issues, discrepancies, communication issues etc)
- Will there be any additional fields when users submit supplementary/corrective reports?

This explores some scope of #117 in depth and informs ticket #178 in parallel

#### Acceptance Criteria

- [x] Who is the best person to speak to about this? **PR: John, Harnav, Adit, Robb T, Jairo as optional***
- [x] Map process of supplementary reports, corrective/supplementary differences, how it was handled in CIIP **<- Not CIIP, SWRS (PR)**
- [x] Ensure BA is in alignment about the decisions we make for supplementary/corrective reports

#### Additional context

Add any other context about the task here.

**Comment by NicoleGovvy** [comment]
@suhafa do the questions you posted in the SME chat cover this ticket then? It seems like it.

**Comment by patriciarussellCAS** [comment]
@NicoleGovvy & @suhafa I think we need an additional ticket for Directors Determination. 
From the internal user role meeting on March 25, IRC confirmed that Supplementary and Corrected reports are a new submission of a report, but that a Director's Determination is more like an attachment or appendix to a report. It's related to this ticket but separate. Also impacted by #178

**Comment by suhafa** [comment]
@patriciarussellCAS good point, we will need a ticket to capture this feature!

**Comment by patriciarussellCAS** [comment]
IRC is planning a process mapping session for this so the ticket it somewhat blocked by this. (April 15 2024)

**Comment by suhafa** [comment]
Had a meeting with John and Emilie where we decided:
- 'Edit report' to be used instead of 'Submit supplementary report'
- It doesn't add any value on the internal side for the external user to indicate whether a report was corrected or supplementary.

There is more discussion to be had about how editing reports takes place. As far as supplementary/corrected reports go, we've made our decision on not having the front end reflect these words. Tagging @mikevespi FYI
